process/coordinator/transactionType.go

```diff
@@ -75,11 +75,15 @@ func (tth *txTypeHandler) ComputeTransactionType(tx data.TransactionHandler) pro
 		return process.MoveBalance
 	}
 
+	funcName := tth.getFunctionFromArguments(tx.GetData())
+	isBuiltInFunction := tth.isBuiltInFunctionCall(funcName)
 	if isAsynchronousCallBack(tx) {
+		if isBuiltInFunction {
```

Comment:
From what I see we can get away with `if isBuiltInFunction {...` outside of this if and remove it from line 100. There is no case where `isBuiltInFunction == true` and we return something else. 
At line 96 we can just have `if !isDestInSelfShard`

Reply:
done